pkg/cmd/experimental/syncgroups/ad/augmented_ldapinterface.go

```diff
@@ -6,7 +6,7 @@ import (
 	"github.com/go-ldap/ldap"
 
 	"github.com/openshift/origin/pkg/auth/ldaputil"
-	ldapinterfaces "github.com/openshift/origin/pkg/cmd/experimental/syncgroups/interfaces"
+	"github.com/openshift/origin/pkg/cmd/experimental/syncgroups/interfaces"
```

Comment:
`ldapinterfaces` so the package is consistent across all imports.  It makes reviewing code easier.